test/sqllogictest/range.slt

```diff
@@ -839,2343 +839,4367 @@ SELECT a::text t, array_agg(v::text ORDER BY v) FROM (
 [1,)
 {"(,1)","[-1,1)"}
 
-
 #
-# int8range
+# int4range union
```

Comment:
Could you add the script you used to generate these cases? I'm hoping it's a python script and we can rerun it if we want to add a new case to these tests, for example a new operator, new type or new value.

Reply:
To make it more clear, probably makes sense to have a separate slt file or at least a start+end delimited block in this file to show which part is script-generated.

Reply:
I sadly was not so principled about it because the script only takes a minute to write; I just toss it away when I'm done. In the future I'll be more diligent about making that easier to share.